batch/batch/worker.py

```diff
@@ -93,10 +94,14 @@ def free(self, port):
             elif e.status == 500 and ("request canceled while waiting for connection" in e.message or
                                       re.match("error creating overlay mount.*device or resource busy", e.message)):
                 log.exception('in docker call, retrying')
+            # DockerError(500, 'Error response from daemon: Conflict. The name * is already in use by container *. You have to remove (or rename) that container to be able to reuse that name.')
+            elif e.status == 500 and "You have to remove (or rename) that container to be able to reuse that name." in e.message:
+                return
             else:
                 raise
         except asyncio.TimeoutError:
-            log.exception('in docker call, retrying')
+            f_name = f.__name__
+            log.exception(f'in docker call {f_name}, retrying', stack_info=True)
```

Comment:
I'd say 'in docker call to {f_name}'.  Also, I think you can just say {f.__name__} in the f-string.